src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -264,7 +251,8 @@ NavHeader.propTypes = {
   placeholderText: PropTypes.string,
   setIsFirstTimeOpen: PropTypes.func,
   getSearchLayout: PropTypes.func,
-  hasSearch: PropTypes.bool
+  hasSearch: PropTypes.bool,
+  onFollowButtonPress: PropTypes.func
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The PropTypes have been updated to include `onFollowButtonPress`. This change seems fine as long as the `onFollowButtonPress` prop is always provided and is a function.
```diff
+ onFollowButtonPress: PropTypes.func
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->